src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -6156,6 +6156,27 @@ MethodTableBuilder::InitMethodDesc(
 #endif // !_DEBUG
         pNewMD->SetSynchronized();
 
+    HRESULT hr = pNewMD->GetCustomAttribute(
```

Comment:
This is non-pay for play. We have intentionally avoided the eager lookup of the attribute in the type loader and burning a bit on MethodDesc for it.

Reply:
Makes sense, I am testing a new version done that way.